about.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/8d0a2aa6d4.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <title>first capstone project</title>
+</head>
+<body>
+    <header>
+        <nav class="nav_bar">
+          <div class="humburger_menu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+          </div> 
+          <a href="#" class="logo"><img src="images/label.png" alt="Logo"></a>
+          <ul class="mobile-menu-list">
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="">concert</a></li>
+            <li><a href="">Ticket</a></li>
+            <li><a href="">Shows</a></li>
+            <li><a href="">News</a></li>
+          </ul>
+          <button type="submit" class="nav-ticket">ticket</button>
+          <ul class="nav-link">
+            <li><a class="nav-link-line" href="./index.html">Home</a></li>
+            <li><a class="nav-link-line" href="#">concert</a></li>
+            <li><a class="nav-link-line" href="#">ticket</a></li>
+          </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="homepage">
+            <div class="homepage-content">
+                <h3 class="hello">"Hello! Party People"</h3>
+                <h1 class="monster_of_rock">Monsters of rock <br> world concert <br> 2023</h1>
+                <p class="long_P">A great concert celebrating multiple talents from various counties <br>  
+                    which is taking place in gotham city on 15th october
+                    please y'all welcome to the greatest show of all time
+                </p>
+                <p class="date">
+                    Please contact us for more information
+                </p>
+                <p class="location">
+                    Email: <a href="">Record2021@gmail.com</a> 
+                </p>
+            </div>
```

Comment:
The styling for the about section can still be improved if the navigation can match that of the behance version instead of just displaying an hamburger menu, the alignment of the content on the headline section are proper (appropriate font sizes and color for the text should be considered too)

YOUR VERSION
![notGood](https://user-images.githubusercontent.com/84930355/225163415-7a8732ba-5165-4c0a-b41a-ebea4db9e425.PNG)



BEHANCE VERSION

![about](https://user-images.githubusercontent.com/84930355/225162765-068b8c1e-4918-449e-be63-c1de88394d12.PNG)

Reply:
- [ ] Again here as requested by the previous reviewer, kindly style your about page to match the given design, kindly fix the overlapping content, and center your content as is in the given design.
#### Note to match the fonts and colors as requested by the previous reviewer

Your design
![image](https://user-images.githubusercontent.com/81932478/225370214-b048419d-b845-49e4-b9eb-4b250a6f562f.png)


Given design
![image](https://user-images.githubusercontent.com/81932478/225369957-853c9e2c-edf9-420a-a99c-daa197606c0c.png)